### PR TITLE
Add additional logging

### PR DESCRIPTION
**Pull request by matt-rich**

### CODE REVIEW

File: core/src/main/java/uk/gov/justice/services/core/jms/JmsSender.java [hunk 1]
```diff
@@ -9,15 +9,21 @@
...
 
     private final JmsDestinations jmsDestinations;
     private final Component destinationComponent;
     private final JmsEnvelopeSender jmsEnvelopeSender;
 
+    @Inject
+    Logger logger;
+
     public JmsSender(final Component destinationComponent, final JmsDestinations jmsDestinations,
                      JmsEnvelopeSender jmsEnvelopeSender) {
```
Comment (mapingo) [inline core/src/main/java/uk/gov/justice/services/core/jms/JmsSender.java]: Not something you changed, but there is a final missing here

File: core/src/test/java/uk/gov/justice/services/core/json/JsonSchemaValidatorTest.java [hunk 2]
```diff
@@ -11,50 +11,62 @@
...
 import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.slf4j.Logger;
 
 /**
  * Unit tests for the {@link JsonSchemaValidator} class.
  */
 @RunWith(MockitoJUnitRunner.class)
 public class JsonSchemaValidatorTest {
 
+    public static final String TEST_SCHEMA_NAME = "test-schema";
```
Comment (mapingo) [inline core/src/test/java/uk/gov/justice/services/core/json/JsonSchemaValidatorTest.java]: doe this need to be public?

File: core/src/main/java/uk/gov/justice/services/core/jms/JmsSender.java (hunk 1 above)
Comment (matt-rich) [inline core/src/main/java/uk/gov/justice/services/core/jms/JmsSender.java]: Done

File: core/src/test/java/uk/gov/justice/services/core/json/JsonSchemaValidatorTest.java (hunk 2 above)
Comment (matt-rich) [inline core/src/test/java/uk/gov/justice/services/core/json/JsonSchemaValidatorTest.java]: Done

**Pull request merged**